package-res/resources/web/prompting/components/StaticAutocompleteBoxComponent.js

```diff
@@ -15,9 +15,57 @@
  *
  */
 
+/**
+ * <h2>The Static Autocomplete Box Component class</h2>
+ *
+ * <p>The StaticAutocompleteBoxComponent renders an autocomplete input box that helps users with the sugestions based on his input</p>
+ *
+ * To use the StaticAutocompleteBoxComponent you should require the appropriate file from common-ui:
+ *
+ * <pre><code>
+ *   require([ 'common-ui/components/StaticAutocompleteBoxComponent' ],
```

Comment:
need replace path to 'common-ui/prompting/components/StaticAutocompleteBoxComponent'